master/types.html

```diff
@@ -1578,7 +1578,7 @@ <h2 id="DOMInterfacesForReflectingSVGAttributes">DOM interfaces for reflecting S
 
 </div>
 
-<p>Some IDL attributes <dfn id="TermReflect">reflect</dfn> the value of a content attribute
+<p>Some IDL attributes <dfn id="TermReflect" data-dfn-type="dfn">reflect</dfn> the value of a content attribute
```

Comment:
I figured reflect must be cononically defined in some other spec as it is a common DOM/HTML term. so didn't add the data-dfn-exports. Should these type of terms be marked up as dfn or just linked to whichever of DOM or HTML that defines it? Same applies to a few other things in SVG like stacking contexts etc.